ddtrace/contrib/internal/azure_eventhubs/utils.py

```diff
@@ -0,0 +1,125 @@
+from typing import Union
+from uuid import UUID
+
+import azure.eventhub as azure_eventhub
+import azure.eventhub.amqp as azure_eventhub_amqp
+
+from ddtrace import config
+from ddtrace.contrib.trace_utils import ext_service
+from ddtrace.ext import SpanTypes
+from ddtrace.ext import azure_eventhubs as azure_eventhubsx
+from ddtrace.internal import core
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.propagation.http import HTTPPropagator
+
+
+def create_context(context_name, pin, operation_name, resource=None):
```

Comment:
can we type all of these functions?

Reply:
Handled here: https://github.com/DataDog/dd-trace-py/pull/14636/commits/c5c621a77d7df37e7bf0281a512aa86d17d1dd23